tour-of-go/exports/comprssion/compression_math.go

```diff
@@ -0,0 +1,11 @@
+package comprssion
+
+import "fmt"
+
+var cam = 928374 * 938643
+
+func campmath() {
```

Comment:
This is a wrong way of naming a function>
According to what I taught you yesterday, please correct this here and in every affected packages.

`campmath()` should be change to `campMath()`


Why?
1. That is two different words. You have to make the two words visible.
2. It is more readable that way.
3. Improves code quality.
4. And it's golang standard for naming functions.